GetFed/GetFed/Controllers/FoodSearchViewController.swift

```diff
@@ -34,6 +41,34 @@ class FoodSearchViewController: UIViewController {
     }
 }
 
+// MARK - UITableViewDataSource & UITableViewDelegate Protocol Implementation
+
+extension FoodSearchViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    func registerFoodResultTableViewCell() {
+        let cell = UINib(nibName: Identity.FoodResultTableViewCell.nibID, bundle: nil)
+        foodTableView.register(cell, forCellReuseIdentifier: Identity.foodSearchResultCell.cellID)
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard let rowCount = searchResults?.results.count else { return 0 }
+        if rowCount == 0 {
+            return 1
+        } else {
+            return rowCount
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: Identity.foodSearchResultCell.cellID, for: indexPath) as? FoodResultTableViewCell else { return UITableViewCell() }
```

Comment:
The else here would be a good example of when to force a crash, as not having a cell means there's a significant issue in your logic. I usually use `fatalError("<description>")`.

Reply:
Added this. Seems like a much better way of handling the else case than returning an empty UITableViewCell object.